packages/contracts-bedrock/tasks/validate-spacers.ts

```diff
@@ -12,10 +16,37 @@ task(
       continue
     }
 
+    // Some files may not have buildInfo (certain libraries). We can safely skip these because we
+    // make sure that everything is accounted for anyway.
     const buildInfo = await hre.artifacts.getBuildInfo(name)
+    if (buildInfo === undefined) {
```

Comment:
Perhaps logging here would be useful as to not have a silent error go without us knowing

Reply:
It's not really an error. Since we do "accounting" based on the lock file it doesn't really matter even if there were an error here.

Reply:
Adding a log anyway